pages/account.js

```diff
@@ -0,0 +1,207 @@
+import React, {Component} from 'react';
+import NavbarBootstrap from '../components/NavbarBootstrap';
+import Container from 'react-bootstrap/Container';
+import Card from 'react-bootstrap/Card';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import Nav from 'react-bootstrap/Nav';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import CarRow from '../components/CarRow';
+import axios from 'axios';
+
+const host = 'http://localhost';
+const port = 3001;
+
+class Account extends Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            isAuthenticated: false,
+            name: '',
+            email: '',
+            failedProfileUpdate: false,
+            succeededProfileUpdate: false,
+        };
+    }
+
+    componentDidMount = () => {
+        const authToken = window.localStorage.getItem('auth_token');
+
+        if (!authToken) {
+            this.setState({ isAuthenticated: false });
+        } else {
+            axios.post(`${host}:${port}/authstatus`, { headers: { 'Authorization' : `Bearer ${authToken}` } }).then(res => {
+                if (res.status === 200) {
+                    const { name, email } = res.data.authData;
+                    this.setState({
+                        isAuthenticated: true,
+                        name,
+                        email,
+                    });
+                }
+            }).catch(err => {});
+        }
+    }
+
+    updateProfile = () => {
+        const { name, email } = this.state;
+
+        axios.post(`${host}:${port}/updateProfile`, {
+            'name': name,
+            'email': email,
+        })
+        .then(response => {
+            const { name, email, token } = response.data;
+            window.localStorage.setItem('auth_token', token);
+            this.setState({ name, email, succeededProfileUpdate: true });
+        })
+        .catch(error => {
+            console.log(error);
+            this.setState({ failedProfileUpdate: true });
+        });
+    }
+
+    createCarRow = () => {
+        let children = [];
```

Comment:
lets rename children to something more meaningful like `vehiclesOwned` or something